File: DFRobot_RGBLCD/python3/Display/Display.py
import sys
sys.path.append('../')
import rgb1602
import time

lcd=rgb1602.RGB1602(16,2)                               #create LCD object,specify col and row

lcd.printout("hello world!")
while True:
  lcd.noDisplay()
  time.sleep(0.5)
  lcd.display()
  time.sleep(0.5)

File: cpp/examples/Display.cpp
#include "../DFRobot_RGBLCD.h"

int main(){
  DFRobot_RGBLCD lcd(16, 2);
  lcd.init();

  lcd.print("Hello World!");
  
  while(1){
    // Turn off the display:
    lcd.noDisplay();
    delay(500);
    // Turn on the display:
    lcd.display();
    delay(500);
  }
  return 0;
}
